### ISSUE TITLE
API doc search box is extremely low contrast

### ISSUE THREAD

**Issue by stevage** [mapbox/mapbox-gl-js-docs#370]
The text in the API search box is just about invisible for me.

![Screen Shot 2020-07-15 at 10 04 06 pm](https://user-images.githubusercontent.com/678344/87542959-2cd31f00-c6e7-11ea-9391-0154c79ce69f.png)

On my main monitor it's poor, but bearable. On my external monitor, it really requires concerted staring to read the text.

Looking at the CSS I can't really figure out where this very pale color comes from, or even how this hint text is constructed exactyl.

**Comment by katydecorah** [comment]
You're right that placeholder text is really hard to see. I'll look into this. Thanks for flagging!
